sample-apps/python-manual-instrumentation-sample-app/app.py

```diff
@@ -144,29 +151,32 @@ def call_http():
     with tracer.start_as_current_span("outgoing-http-call") as span:
 
         # Demonstrates setting an attribute, a k/v pairing.
-        span.set_attribute("operation.name", "outgoing-http-call")
+        span.set_attribute("language", "python-manual-instrumentation")
+        span.set_attribute("signal", "trace")
 
         # Demonstrating adding events to the span. Think of events as a primitive log.
         span.add_event("Making a request to https://aws.amazon.com/")
-        with tracer.start_as_current_span("amazon-request") as child:
-            requests.get("https://aws.amazon.com/")
+        requests.get("https://aws.amazon.com/")
 
-            print("updating bytes sent & latency time...")
-            update_total_bytes_sent()
-            update_latency_time()
+        print("updating bytes sent & latency time...")
+        update_total_bytes_sent()
+        update_latency_time()
 
-            return app.make_response(
-                convert_otel_trace_id_to_xray(
-                    trace.get_current_span().get_span_context().trace_id
-                )
+        return app.make_response(
+            convert_otel_trace_id_to_xray(
+                trace.get_current_span().get_span_context().trace_id
```

Comment:
Shouldn't an x-ray ID generator be used  instead? Or is this the way that python has to handle it?

Reply:
The x-ray ID generator is added to the tracer config [here](https://github.com/aws-observability/aws-otel-community/blob/0417a3bdea4952b2374624da4b20a80b8e67fc22/sample-apps/python-manual-instrumentation-sample-app/app.py#L77)